include/_mgp_mock.py

```diff
@@ -0,0 +1,183 @@
+import typing
+
+import networkx as nx
+
+
+class ImmutableObjectError(Exception):
+    pass
+
+
+class LogicErrorError(Exception):
+    pass
+
+
+class Vertex:
+    __slots__ = ("_id", "_graph")
+
+    def __init__(self, id: int, graph: nx.MultiDiGraph) -> None:
+        if not isinstance(id, int):
+            raise TypeError(f"Expected 'int', got '{type(id)}'")
+
+        if not isinstance(graph, nx.MultiDiGraph):
+            raise TypeError(f"Expected 'networkx.classes.multidigraph.MultiDiGraph', got '{type(graph)}'")
+
+        if not graph.has_node(id):
+            raise IndexError(f"Unable to find vertex with ID {id}.")
+
+        self._id = id
+        self._graph = graph
+
+    @property
+    def id(self) -> int:
+        return self._id
+
+    @property
+    def graph(self) -> nx.MultiDiGraph:
+        return self._graph
+
+    @property
+    def labels(self) -> typing.List[int]:
+        return self._graph.nodes[self._id]["label"].split(":")
+
+    def is_valid(self) -> bool:
+        return self._graph.has_node(self._id)
+
+    def add_label(self, label: str) -> None:
+        if nx.is_frozen(self._graph):
+            raise ImmutableObjectError("Cannot modify immutable object.")
+
+        self._graph.nodes[self._id]["label"] += f":{label}"
+
+    def remove_label(self, label: str) -> None:
+        if nx.is_frozen(self._graph):
+            raise ImmutableObjectError("Cannot modify immutable object.")
+
+        labels = self._graph.nodes[self._id]["label"]
+        if labels.startswith(f"{label}:"):
+            labels = "\n" + labels  # pseudo-string starter
+            self._graph.nodes[self._id]["label"] = labels.replace(f"\n{label}:", "")
+        elif labels.endswith(f":{label}"):
+            labels += "\n"  # pseudo-string terminator
+            self._graph.nodes[self._id]["label"] = labels.replace(f":{label}\n", "")
```

Comment:
Why do you need pseudo-string starter and pseudo-string terminator?

Reply:
@antoniofilipovic I’m using the same Memgraph-to-nx graph “translation” as @as51340 in his [PR](https://github.com/memgraph/gqlalchemy/pull/215/files#diff-c88326fe380799b9832efa8b85885c70032853e96e73ab709ec8b84267519daf) on gqlalchemy, which stores labels as a string like this: `"Label1:Label2:Label3"`.
@Josipmrden There are three cases here wrt removal:
* first label: remove `{label}:`
* last label: remove `:{label}`
* default case: replace `:{label}:` with `:`.

@as51340 The pseudo-starter and -terminator cover the edge case where the first/last label is a substring of another label. For example, if the labels are `"Graph:DiGraph:Label3"` and I want to delete `Graph`, blindly removing `Graph:` would change the labels to `"Di:Label3"`.

Reply:
makes sense

Reply:
Just put "label" as a constant.

Reply:
@antepusic I would still use here set, as it is easier to maintain code

Reply:
This was changed so I will resolve this